.github/workflows/cd.yml

```diff
@@ -0,0 +1,61 @@
+name: Continuous Delivery
+on:
+  push:
+    tags:
+    - 'v*'
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Setup .NET Core 2.1
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '2.1.607'
+    - name: Install EF CLI
+      shell: cmd
+      run: dotnet.exe tool install --global dotnet-ef --version 3.0.0
+    - name: Create Release
+      shell: cmd
+      run: dotnet.exe publish -o "%GITHUB_WORKSPACE%\release" -r linux-musl-x64 -c Release src\Ztm.WebApi
+    - name: Create Migration Scripts for Main Database
+      shell: cmd
+      env:
+        ZTM_MAIN_DATABASE: Host=127.0.0.1;Database=postgres;Username=postgres
+      run: dotnet.exe ef migrations script -o "%GITHUB_WORKSPACE%\migration\migration.sql" -i -p src\Ztm.Data.Entity.Postgres
+    - name: Archive Release Artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: release
+        path: release
+    - name: Archive Migration Artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: migration
+        path: migration
+  docker:
+    name: Build Docker Image
+    needs: build
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Get Release Artifact
+      uses: actions/download-artifact@v1
+      with:
+        name: release
+        path: release
+    - name: Fix File Permission
+      run: chmod +x release/Ztm.WebApi
+    - name: Login Docker
+      uses: azure/docker-login@v1
+      with:
+        login-server: docker.pkg.github.com
+        username: $GITHUB_ACTOR
```

Comment:
I think better to use `secrets` so it match with `password`.

Reply:
Not sure what field from `secrets` you mean. I updated to `${{ github.repository }}` instead.